src/DeclarationScope.ts

```diff
@@ -194,7 +194,7 @@ export class DeclarationScope {
     }
     const { expression } = node.name;
 
-    if (ts.isIdentifier(expression)) {
+    if (ts.isIdentifier(expression) || ts.isStringLiteral(expression)) {
```

Comment:
Actually we rather want to skip this, instead of running it through the branch. By definition, there is no way a string literal will refer to anything you have imported.

Also, it is probably the same for numbers and other literals as well? Some tests to cover this would be nice as well.